modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/Model.scala

```diff
@@ -66,9 +75,10 @@ object Model {
     // Generic update. It will probably become useless if we have a special Event for every case.
     case class SequenceUpdated(view: SequencesQueue[SequenceView]) extends SeqexecModelUpdate
 
-    // TODO: msg should be LogMsg bit it does IO when getting a timestamp, it
+    // TODO: msg should be LogMsg but it does IO when getting a timestamp, it
```

Comment:
This comment can be removed. It's not doing any IO, the timestamp is passed as a parameter.